helm/config/nginx.conf

```diff
@@ -0,0 +1,68 @@
+#user  nginx;
+worker_processes  4;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /tmp/nginx.pid;
+load_module modules/ngx_http_js_module.so;
+load_module /otel/otel_ngx_module.so;
+
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    opentelemetry_config /otel/otel-nginx.toml;
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format json escape=json 
```

Comment:
consult with Roee, on geoserver i used the default log_format of infra nginx, I think you should use it too, on future we will remove it also from pycsw the specific format